docs/test/replication-controller-list.md

```diff
@@ -0,0 +1,39 @@
+# Replication controller list
+
+## Prerequisites
+Cluster with few deployed applications, where: 
+
+* at least one is valid
+* at least one is invalid (for example with wrong image) 
+
+## Expected result
+Page displays correctly, especially:
+
+* cards are displayed correctly even when page is scaled down/up
+* application details are valid and matching data from `kubectl` command
+* logs menu display correct pods information
+* all application actions are available
+* invalid applications have warning symbols and error messages
+
+## Specific factors
+
+| Factor                               | 1                                                | 2                                             | 3                                        | 4                                       | 5                                                                 | 6                                                                   | Comment |
+|--------------------------------------|--------------------------------------------------|-----------------------------------------------|------------------------------------------|-----------------------------------------|-------------------------------------------------------------------|---------------------------------------------------------------------|---------|
+| Replication controller count on page | 1 replication controller on wide page            | 1 replication controller on narrow page       | 3 replication controller on on wide page | 3 replication controller on narrow page | Any other positive number of replication controllers on wide page | Any other positive number of replication controllers on narrow page |         |
+| Replication controller status        | Valid                                            | Invalid (for example with not-existing image) |                                          |                                         |                                                                   |                                                                     |         |
```

Comment:
RC status  | valid | error    | Comment: error increases height of card

Less text is better, I think